packages/node_modules/@ciscospark/plugin-authorization-browser-first-party/test/unit/spec/authorization.js

```diff
@@ -225,5 +225,48 @@ browserOnly(describe)(`plugin-authorization-browser-first-party`, () => {
           });
       });
     });
+
+    describe(`#_cleanUrl()`, () => {
+      it(`removes the state parameter when it has no keys`, () => {
+        const spark = makeSpark(undefined, undefined, {
+          credentials: {
+            clientType: `confidential`
+          }
+        });
+        const location = {
+          query: {
+            code: `code`,
+            state: {
+              csrf_token: `token`
+            }
+          }
+        };
+        sinon.spy(spark.authorization, `_cleanUrl`);
+        spark.authorization._cleanUrl(location);
+        assert.called(spark.getWindow().history.replaceState);
+        assert.equal(spark.getWindow().location.href, ``);
+      });
+
+      it(`keeps the parameter when it has keys`, () => {
+        const spark = makeSpark(undefined, undefined, {
+          credentials: {
+            clientType: `confidential`
+          }
+        });
+        const location = {
+          query: {
+            code: `code`,
+            state: {
+              csrf_token: `token`,
+              key: `value`
+            }
+          }
+        };
+        sinon.spy(spark.authorization, `_cleanUrl`);
+        spark.authorization._cleanUrl(location);
+        assert.called(spark.getWindow().history.replaceState);
+        assert.isDefined(spark.getWindow().location.href);
```

Comment:
This confirms that `_cleanUrl()`. was called, but I don't see where it confirms the intended behavior